index.html

```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css" />
```

Comment:
Please correct the link as there is no css folder in your branch.

Reply:
[OPTIONAL] - I think you can make use of emojis and gifs to make the code review more human-friendly 👽🤖.